tasks/dist.js

```diff
@@ -0,0 +1,24 @@
+const gulp = require('gulp')
```

Comment:
why is gulp in the top level? this should only be used by people messing with the site